src/runtime/audience-action-flow.js

```diff
@@ -92,6 +109,11 @@ export class AudienceActionFlow {
       this.handleEntitlementsResponse_.bind(this)
```

Comment:
Should we update this to listen on `CompleteAudienceActionResponse` now that that is being passed back?

Reply:
I have a follow-up PR for this :)